src/main/java/com/easysplit/ess/transactions/domain/sql/TransactionsQueries.java

```diff
@@ -29,6 +31,10 @@ private TransactionsQueries() {
     public static final String CREATED_DATE_COLUMN = "CREATED_DATE";
     public static final String UPDATED_BY_COLUMN = "UPDATED_BY";
     public static final String UPDATED_DATE_COLUMN = "UPDATED_DATE";
+    public static final String DEBTGUID_COLUMN = "DEBTGUID";
```

Comment:
Let's remove this constants since they are already or should be defined in the DebtsQueries.java class, not need to keep them here aswell